### PR TITLE
@feature/complete flow

### PR DESCRIPTION
**Pull request by ukirderohit**

### CODE REVIEW

File: packages/im-base/src/constants/mockDBResponse.json [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          "url": "https://i.ytimg.com/vi/DLX62G4lc44/maxresdefault.jpg",
+          "width": 1280,
+          "height": 720
+        }
+      },
+      "genre": "Thriller",
+      "hotspots": [
+        {
+          "id": 1,
+          "name": "Test",
```
Comment (pranay2401) [inline packages/im-base/src/constants/mockDBResponse.json:40]: update mockData to contain hotspots. overlays and triggers as objects and not lists

File: packages/im-base/src/readers/Movie.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// Lodash
+import _property from "lodash/property";
+
+const id = _property("id");
+const title = _property("title");
+const name = _property("name");
+const releaseDate = _property("releaseDate");
```
Comment (pranay2401) [inline packages/im-base/src/readers/Movie.js:7]: releaseDate is not an attribute for movie.
Add data for hotspots, overlays, triggers, editorId, publishedAt, createdAt etc.

File: packages/im-components/src/organisms/seeker/seeker.module.scss [hunk 3]
```diff
@@ -5,14 +5,25 @@ $contrast: -5%;
...
 .seekerWrapper {
   margin: 0 auto;
-  padding: 3rem;
 
   .reactSeeker {
     margin-bottom: 1rem;
     background: rgba(0, 0, 0, 0.1);
     width: 100%;
     height: 27rem;
   }
```
Comment (pranay2401) [inline packages/im-components/src/organisms/seeker/seeker.module.scss:14]: add lines between css elements

File: packages/im-ott/src/App.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router } from "react-router-dom";
 // Components
 import { ErrorBoundary } from "@sentry/react";
 import AppLoader from "imcomponents/molecules/appLoader";
+import { ToastContainer } from "imcomponents/atoms/toaster";
 // import SocketIoTest from "./SocketIoTest";
```
Comment (pranay2401) [inline packages/im-ott/src/App.js:7]: remove commented code

File: packages/im-studio/src/graphql/queries.js [hunk 5]
```diff
@@ -1,9 +1,117 @@
...
+      hotspots
+      overlays
+      triggers
+      interactiveData
+    }
   }
 `;
 
-export { HELLO_WORLD };
+const QUERY_MOVIE_ID = gql`
```
Comment (pranay2401) [inline packages/im-studio/src/graphql/queries.js:60]: Should be QUERY_MOVIE_BY_ID

File: packages/im-studio/src/pages/createVideo/CreateVideo.js [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+              <EditTab tabdata={movieData} history={history} />
+            </TabPane>
+            <TabPane tab="Hotspots" key={"#2"}>
+              <Hotspots tabdata={movieData} history={history} />
+            </TabPane>
+            <TabPane tab="Overlays" key={"#3"}>
+              <OverlaysTab tabdata={movieData} history={history} />
+            </TabPane>
+            <TabPane tab="Time Triggers" key={"#4"}>
+              <TimeTriggersTab />
```
Comment (pranay2401) [inline packages/im-studio/src/pages/createVideo/CreateVideo.js]: Why is tabData not passed to TimeTriggers and  Publish Tabs?

File: packages/im-studio/src/pages/editVideo/EditVideo.js [hunk 7]
```diff
@@ -18,76 +16,76 @@ import OverlaysTab from "../../organisms/overlaysTab";
...
-                </div>
-            </div>
+  return (
+    <div className={styles.container}>
+      <div className={styles.content}>
+        <div className={styles.player}>
+          <Player
+            isHost={false}
+            videoUrl={`http://www.youtube.com/watch?v=${movie.mId}`}
+          ></Player>
```
Comment (pranay2401) [inline packages/im-studio/src/pages/editVideo/EditVideo.js:65]: What is the difference between EditVideo page and Create Video page?

File: packages/im-studio/src/routes/AppRoutes.js [hunk 8]
```diff
@@ -31,6 +32,7 @@ const AppRoutes = () => {
         </Route>
         <Route exact path="/dashboard" component={Dashboard} />
         <Route exact path="/upload" component={Upload} />
+        {/* <Route exact path="/video/:videoId/create" component={CreateVideo} /> */}
```
Comment (pranay2401) [inline packages/im-studio/src/routes/AppRoutes.js]: why two separate pages and components if not using CreateVideo page?

File: yarn.lock [hunk 9]
```diff
@@ -36,6 +36,42 @@
     lodash "^4.17.21"
     resize-observer-polyfill "^1.5.0"
 
+"@apollo/client@^3.4.3":
```
Comment (pranay2401) [inline yarn.lock:39]: should this file be pushed?

**Review by pranay2401 — CHANGES_REQUESTED** [review]
Please incorporate comments

File: packages/im-base/src/constants/mockDBResponse.json (hunk 1 above)
Comment (ukirderohit) [reply packages/im-base/src/constants/mockDBResponse.json:40]: we won't need mock data any more for this one

File: packages/im-ott/src/App.js (hunk 4 above)
Comment (ukirderohit) [reply packages/im-ott/src/App.js:7]: This was added intentionally to test socket.io, lets keep it there for a while until we decide not to go with socket and can be removed later

File: yarn.lock (hunk 9 above)
Comment (ukirderohit) [reply yarn.lock:39]: yeah, always. It's better to have lock files

File: packages/im-components/src/organisms/player/Player.js [hunk 10]
```diff
@@ -188,14 +187,12 @@ const Player = (props) => {
...
   videoUrl: PropTypes.string,
-  debug: PropTypes.bool,
   isHost: PropTypes.bool,
 };
 
 Player.defaultProps = {
   className: undefined,
-  videoUrl: "https://www.youtube.com/watch?v=bxnYFOixIoc",
-  debug: false,
+  videoUrl: "",
```
Comment (pranay2401) [inline packages/im-components/src/organisms/player/Player.js]: Use empty string constant

**Review by pranay2401 — CHANGES_REQUESTED** [review]
Please change

**Pull request merged**